pkgs/applications/blockchains/dogecoin.nix

```diff
@@ -7,18 +7,17 @@
 with lib;
 stdenv.mkDerivation rec {
   name = "dogecoin" + (toString (optional (!withGui) "d")) + "-" + version;
-  version = "1.14.2";
+  version = "1.14.3";
 
   src = fetchFromGitHub {
     owner = "dogecoin";
     repo = "dogecoin";
     rev = "v${version}";
-    sha256 = "1gw46q63mjzwvb17ck6p1bap2xpdrap08szw2kjhasa3yvd5swyy";
+    sha256 = "15yjzyvgic96pybadfk37zb032xvra0v37iphbnh15dci2fd934j";
   };
 
-  nativeBuildInputs = [ pkg-config autoreconfHook ];
-  buildInputs = [ openssl db5 openssl util-linux
-                  protobuf boost zlib miniupnpc libevent ]
+  nativeBuildInputs = [ pkg-config autoreconfHook util-linux ];
+  buildInputs = [ openssl db5 openssl protobuf boost zlib miniupnpc libevent ]
```

Comment:
```suggestion
  buildInputs = [ openssl db5 rotobuf boost zlib miniupnpc libevent ]
```

Reply:
The [README](https://github.com/dogecoin/dogecoin/blob/master/doc/build-unix.md#dependencies) states that `libssl` is required.